src/lokijs.js

```diff
@@ -359,6 +359,14 @@
           }
         }
         return false;
+      },
+
+      $size: function (a, b) {
+        if (!Array.isArray(a)) {
+          return false;
```

Comment:
oh ok - well as I understand the operator `$size` it tests an arbitrary number against the length of an existing array, so this is either true or false. Well in this plane of existence at least. So yeah, we're good.